src/menu.css.ts

```diff
@@ -31,15 +37,17 @@ export const menuContainer = style([
         top,
         bottom,
         left,
+        right,
         width,
```

Comment:
Should we add a prop to control the menu height? 🤔

Reply:
After discussion offline, we may think about this for a second iteration